addons/mail/controllers/discuss.py

```diff
@@ -432,6 +432,20 @@ def mail_thread_data(self, thread_model, thread_id, request_list, **kwargs):
         thread = request.env[thread_model].with_context(active_test=False).search([('id', '=', thread_id)])
         return thread._get_mail_thread_data(request_list)
 
+    def _mail_thread_messages_filter(self):
+        """ Allows to add AND clause on mail.message query to filter further the message returned in the chatter.
```

Comment:
(just to follow the [pep](https://peps.python.org/pep-0257/))
```suggestion
        """Allows to add AND clause on mail.message query to filter further the message returned in the chatter.
```

Reply:
Changed, thanks.